src/features/emails/components/EmailEditor/tools/LibraryImage/LibraryImageEditableBlock.tsx

```diff
@@ -0,0 +1,55 @@
+import { FC, useState } from 'react';
+
+import FileLibraryDialog from 'features/files/components/FileLibraryDialog';
+import { LibraryImageData } from './types';
+
+type Props = {
+  data: LibraryImageData;
+  onChange: (data: LibraryImageData) => void;
+  orgId: number;
+};
+
+const LibraryImageEditableBlock: FC<Props> = ({
+  data: initialData,
+  onChange,
+  orgId,
+}) => {
+  // Start in "selecting" state if the block was just added
+  const [selecting, setSelecting] = useState(!initialData.fileId);
+  const [data, setData] = useState(initialData);
+
+  // TODO: Add way of handling alt text
+  return (
+    <>
+      {data.url && (
+        <img
```

Comment:
I just wondered is there a reason why it doesn't use next image? Maybe it is because the provider only make Editor.js to access react things??

Reply:
To be honest I don't want to use the Nextjs `Image` component because I think it has some problems that I haven't been able to resolve. We have to think about how to solve this anyway, because the images need to be accessible (and trackable) from email clients, not just the browser, so I don't want to try and solve that now.

Reply:
Aha okay! mmm then maybe some codes that can skip the eslint warning about <img> here?